contracts/UsingWitnet.sol

```diff
@@ -6,84 +6,68 @@ pragma experimental ABIEncoderV2;
+  /// parties) before this method returns `true`.
+  /// @param _id The unique identifier of a request that has been previously sent to the WitnetRequestBoard.
+  /// @return A boolean telling if the request has been already resolved or not. Returns `false` if called after destroying the result (i.e. `destroyResult(uint256 _id).
   function witnetCheckRequestResolved(uint256 _id) internal view returns (bool) {
     // If the result of the data request in Witnet is not the default, then it means that it has been reported as resolved.
     return witnet.readDrTxHash(_id) != 0;
   }
 
-  /**
-   * @notice Retrieves result of previously posted DR, and removes it from storage.
-   * @param _id The unique identifier of a previously posted data request.
-   * @return The result of the DR
-  **/
+  /// @notice Retrieves result of previously posted DR, and removes it from storage.
+  /// @param _id The unique identifier of a previously posted data request.
+  /// @return The result of the DR
   function witnetDestroyResult(uint256 _id) internal returns (WitnetData.Result memory) {
     return Witnet.resultFromCborBytes(witnet.destroyResult(_id));
   }
 
-  /**
-   * @notice Estimate the reward amount.
-   * @param _gasPrice The gas price for which we want to retrieve the estimation.
-   * @return The reward to be included for the given gas price.
-  **/
+  /// @notice Estimate the reward amount.
+  /// @param _gasPrice The gas price for which we want to retrieve the estimation.
+  /// @return The reward to be included for the given gas price.
   function witnetEstimateGasCost(uint256 _gasPrice) internal view returns (uint256) {
     return witnet.estimateGasCost(_gasPrice);
   }
 
- /**
-  * @notice Send a new request to the Witnet network with transaction value as result report reward.
-  * @param _request An instance of the `WitnetRequest` contract.
-  * @return Sequencial identifier for the request included in the WitnetRequestBoard.
-  **/
+  /// @notice Send a new request to the Witnet network with transaction value as result report reward.
+  /// @param _request An instance of the `WitnetRequest` contract.
+  /// @return Sequencial identifier for the request included in the WitnetRequestBoard.
   function witnetPostRequest(WitnetRequest _request) internal returns (uint256) {
     return witnet.postDataRequest{value: msg.value}(address(_request));
   }
 
-  /**
-   * @notice Read the result of a resolved request.
-   * @param _id The unique identifier of a request that was posted to Witnet.
-   * @return The result of the request as an instance of `Result`.
-  **/
+  /// @notice Read the result of a resolved request.
+  /// @param _id The unique identifier of a request that was posted to Witnet.
+  /// @return The result of the request as an instance of `Result`.
   function witnetReadResult(uint256 _id) internal view returns (WitnetData.Result memory) {
```

Comment:
```suggestion
  function _witnetReadResult(uint256 _id) internal view returns (WitnetData.Result memory) {
```

Reply:
This change is postponed to `0.4.x` as it would imply breaking changes on the price feed examples, for instance.